src/components/Messages/MessagePreview.jsx

```diff
@@ -74,7 +74,10 @@ const MessagePreview = ({ message, folderType }) => {
               sx={{ padding: '10px' }}
             >
               {messageInfo.map((info, index) => (
-                <Grid item xs={info.xs_value} sx={{ opacity: message.readStatus ? '0.5' : '1' }}
+                <Grid
+                  item
+                  xs={info.xs_value}
+                  sx={{ opacity: message.readStatus ? '0.5' : '1' }}
                   key={info + String(index)}
```

Comment:
FYI this key was updated since replies to messages were running into key errors with title.

Reply:
No problem, any bug fix along the way is more than welcomed. Feel free to use anything from that message object to use as the unique key or to create a unique key to be used.